tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -1003,6 +1003,34 @@ TYPED_TEST(
 		EXPECT_NEAR(c[i] * b[i], a[i], get_epsilon<TypeParam>());
 }
 
+TYPED_TEST(LinalgBackendEigenNonIntegerTypesTest, SGVector_log)
+{
+	const index_t len = 4;
+	TypeParam data[] = {1, 2, 3, 4};
+	SGVector<TypeParam> a(data, len, false);
```

Comment:
This is not what I wrote...
```suggestion
	SGVector<TypeParam> a{1, 2, 3, 4};
```

Reply:
I followed the other examples, I will change it now

Reply:
Ah, that is odd... Anyway, what I showed you here is the correct way to do it, because we have a constructor just for this!

Reply:
Cool I never knew that, It would be quite handy when I add the element-wise divison